config/locales/server.en.yml

```diff
@@ -143,6 +143,9 @@ en:
         creative:
           name: Creative
           description: "AI Bot with no external integrations specialized in creative tasks"
+        dall_e3:
```

Comment:
Minor nit, but only here you have called it `dall_e3`, everywhere else is `dall_e`

Reply:
Not sure if this will affect translations etc.

Reply:
its a tough one, I think I like the command being called dall e cause it has access to 2/3 potentially but the persona is very much tuned for 3 ... not a hill

Reply:
Oh right yes I see the distinction now, I thought it was just an inconsistency. Sounds fine :+1: